apps/meeting/src/utils/api.ts

```diff
@@ -27,21 +27,25 @@ interface GetAttendeeResponse {
   name: string;
 }
 
-export async function fetchMeeting(
```

Comment:
fetch sounds like a `GET` request but it's actually a `POST` so I rename it to `createMeetingAndAttendee`